doc/HOW_THINGS_WORK.md

```diff
@@ -33,24 +33,6 @@ Chef Server has 3 different "search_queue_mode"s:
 This configurable controls how the chef_index module sends requests to
 the search index. The default is `rabbitmq`.
 
-### Rabbitmq
-
-```
-  +--------+    +----------+    +------------------+    +------+
-  | erchef | -> | rabbitmq | <- | opscode-expander | -> | Solr |
-  +--------+    +----------+    +------------------+    +------+
-```
-
-In the rabbitmq search_queue_mode, erchef places the object to be
-indexed on a rabbitmq queue. It them moves forward with the request
-and returns a response to the user.  Another service named
-`opscode-expander` reads the object from the queue, expands it (see
-Document Expansion for details), and posts it to solr. Note that a
-failure to write to the search index will not trigger a failure of the
-API request that wrote the data since it happens asynchronously via a
-queue. However, a failure to place the item on rabbitmq will cause a
-500 error.
-
```

Comment:
Maybe we could keep this but move it to a section about older versions or something.